main/traffic.go

```diff
@@ -174,7 +174,7 @@ func convertMetersToFeet(meters float32) float32 {
 
 func cleanupOldEntries() {
 	for icao_addr, ti := range traffic {
-		if stratuxClock.Since(ti.Last_seen).Seconds() > 60 { // keep it in the database for up to 60 seconds, so we don't lose tail number, etc...
+		if stratuxClock.Since(ti.Last_seen).Seconds() > 30 { // keep it in the database for up to 30 seconds, so we don't lose tail number, etc...
```

Comment:
Any specific reason why the cutoff value needs to be 30 on the server as well? I'd rather keep it at 60, but only send the traffic to the radar/EFB for ~25 seconds, as is done already (or mode-S-only targets for 6 seconds)

Reply:
In order not to consternate users. But you can also leave it, but I do not really understand the 60 secs time ...